packages/mobile/src/screens/edit-track-screen/EditTrackModalScreen.tsx

```diff
@@ -39,10 +43,25 @@ export const EditTrackModalScreen = () => {
 
   const handleSubmit = useCallback(
     (metadata: ExtendedTrackMetadata) => {
-      dispatch(editTrack(id, metadata))
+      if (track?.is_unlisted === true && metadata.is_unlisted === false) {
+        dispatch(
+          setVisibility({
+            drawer: 'ReleaseNowConfirmation',
+            visible: true,
+            data: {
+              trackId: id,
+              handleConfirm: () => {
+                dispatch(editTrack(id, metadata))
```

Comment:
Anyone know if we're doing a bad here by storing a function in redux? @dylanjeffers I have a recollection of that being an anti-pattern.

Is there a better way to do this?